api/src/main/java/com/codeforcommunity/api/ITeamsProcessor.java

```diff
@@ -1,7 +1,16 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
+import com.codeforcommunity.dto.team.*;
```

Comment:
Can you make sure everything is explicitly imported and not with a *? You can make sure IntelliJ doesn't do this automatically in the settings somewhere

Reply:
Preferences > Editor > Code Style > Java > Imports: Change number of lines for * to 999